components/Login.tsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+
+export const Login: React.FC = () =>(
```

Comment:
Purge this as it has nothing to do with this PR. Also, why is it in English again, please be consistent.